.gitlab/rules.yml

```diff
@@ -0,0 +1,34 @@
+.pr_condition:
+  rules:
+    - if: $CI_COMMIT_BRANCH == "develop" || $CI_COMMIT_BRANCH == "master" || $CI_COMMIT_TAG
```

Comment:
Can you add a comment explaining those conditions? Some of them are not 100% obvious to me

Reply:
I added some comments, tell me if that helps